src/app/pages/home-page/home-page.component.scss

```diff
@@ -19,11 +19,22 @@
     height: 100%;
   }
 
-  &__left,
+  &__left {
```

Comment:
maybe we just can use:
max-width: calc(#{$large-desktop} / 2);
and remove mediaquery?

Reply:
yep, I agree with you